README.md

```
@@ -98,78 +62,57 @@ Or install directly from the cask:
+  --url http://localhost:8090/api/v1/servers/time/tools/get_current_time \
+  --header 'Accept: application/json, application/problem+json' \
+  --header 'Content-Type: application/json' \
+  --data '{
+  "timezone": "Europe/Warsaw"
+}'
 ```
 
 API docs will be available at [http://localhost:8090/docs](http://localhost:8090/docs).
 
-## Deploy Anywhere
+## 💡 Why `mcpd`? 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)